ci/recipe/meta.yaml

```
@@ -23,6 +23,7 @@ requirements:
     - pip
     - dask-core {{ dask_version }}
     - versioneer =0.28
+    - setuptools >=62.6
     - tomli
```

Small change to roughly match up with the other Dask recipes

```suggestion
    - tomli # [py<311]
```